enterprise/frontend/src/embedding-sdk/components/public/InteractiveDashboard/ConnectedDashboard.tsx

```diff
@@ -127,19 +130,41 @@ const ConnectedDashboardInner = ({
 
 export const ConnectedDashboard = connector<
   ComponentType<ConnectedDashboardProps & ReduxProps>
->(({ dashboardId: initId, ...rest }) => {
-  const { id: dashboardId, isLoading } = useValidatedEntityId({
+>(({ dashboardId: initialDashboardId, ...rest }) => {
+  const { id: resolvedDashboardId, isLoading } = useValidatedEntityId({
     type: "dashboard",
-    id: initId,
+    id: initialDashboardId,
   });
 
+  const errorPage = useSdkSelector(getErrorPage);
```

Comment:
there's already this data in the store, anytime we try to load a dashboard with the wrong ID, this property will be populated.